PerformanceNavigationTiming for User Agent Launch/security-privacy.md

```diff
@@ -0,0 +1,97 @@
+Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+# 2. Questions to Consider
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+It exposes a new field on the PerformanceNavigationTiming object to allow developers to discern if the page load occured while the user agent was in a non-optimal performance state.
+
+## 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes.
+
+## 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+
+It does not deal with such information.
+
+## 2.4. How do the features in your specification deal with sensitive information?
+
+It does not deal with sensitive information.
+
+## 2.5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+## 2.6. Do the features in your specification expose information about the underlying platform to origins?
+
+No.

```

Comment:
I think part of what this question is getting at is whether there are fingerprinting risks, which seems relevant with this feature. The Privacy section in the explainer addresses that appropriately -- maybe it's worth referencing or summarizing that analysis here?